### ISSUE TITLE
TypeError: Unhandled leaf_type: 5392

### ISSUE THREAD

**Issue by inxbus** [volatilityfoundation/volatility3#539]
hello
cmd: python3 ./pdbconv.py -f  ./symbols/win32k.pdb/ED706A38659240A066E6FB19B994BAAA1/win32k.pdb -o ./json_file/win32k.json
error:
Traceback (most recent call last):ading TPI layer
  File "./pdbconv.py", line 1027, in <module>
    converted_json = convertor.get_json()
  File "./pdbconv.py", line 567, in get_json
    self.read_necessary_streams()
  File "./pdbconv.py", line 335, in read_necessary_streams
    self.read_tpi_stream()
  File "./pdbconv.py", line 343, in read_tpi_stream
    type_references = self._read_info_stream(2, "TPI", self.types)
  File "./pdbconv.py", line 396, in _read_info_stream
    output, consumed = self.consume_type(module, offset, length)
  File "./pdbconv.py", line 802, in consume_type
    subfield, sub_consumed = self.consume_type(module, sub_offset, sub_length)
  File "./pdbconv.py", line 827, in consume_type
    raise TypeError(f"Unhandled leaf_type: {leaf_type}")
TypeError: Unhandled leaf_type: 5392

my version
pip3 show volatility3
Name: volatility3
Version: 1.1.1
Summary: Memory forensics framework
Home-page: https://github.com/volatilityfoundation/volatility3/
Author: Volatility Foundation
Author-email: volatility@volatilityfoundation.org
License: VSL
Location: /usr/local/lib/python3.8/dist-packages/volatility3-1.1.1-py3.8.egg
Requires: pefile
Required-by: 

how can i make a json file by DIA(Debug Interface Access SDK) tool?

**Comment by ikelos** [comment]
We don't have any public tools to create these using DIA, pdbconv is currently the official tool for doing so.  This was designed specifically for handling the kernel which doesn't use C++ types (whereas win32k does), which is why supposed isn't present at the moment.  This is the same as #516, so please subscribe to that issue for updates.  I'm going to close this as a duplicate...

**Comment by ikelos** [comment]
Sorry, this is actually a duplicate of #182.  516 is about new types added to the PDB format.
